sec-theia-docker/README.md

```diff
@@ -0,0 +1,59 @@
+# Security layer for theia-ide
+
+[theia-ide](https://hub.docker.com/r/theiaide/theia) is a powerful mechamism for remote development in your servers. The problem is that it has not authentication mechanism, so if anyone knows your server and port, your code will be exposed.
```

Comment:
typo - `mechamism` should be `mechanism`.

Reply:
typo - `the problem is that it has not authentication mechanism` should be `the problem is that it has no authentication mechanism` (**not** to **no**)